src/features/content-explorer/content-explorer/ContentExplorerIncludeSubfolders.js

```diff
@@ -0,0 +1,74 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { injectIntl, FormattedMessage } from 'react-intl';
+import Checkbox from '../../../components/checkbox/Checkbox';
+import Tooltip from '../../../components/tooltip';
+import Toggle from '../../../components/toggle';
+
+import messages from '../messages';
+import IconInfo from '../../../icons/general/IconInfo';
+
+const ContentExplorerIncludeSubfolders = ({
+    isFolderPresent,
+    handleIncludeSubfoldersToggle,
+    handleSelectAllClick,
+    hideSelectAllCheckbox,
+    isSelectAllChecked,
+    noFoldersSelected,
+    numOfSelectedItems = 0,
+    isToggleDisabled,
+}) => {
+    const getTooltipMessage = () => {
+        if (numOfSelectedItems > 1) {
+            return <FormattedMessage {...messages.includeSubfoldersMultipleFoldersSelected} />;
+        }
+        if (numOfSelectedItems === 1 && noFoldersSelected) {
+            return <FormattedMessage {...messages.includeSubfoldersNoFoldersSelected} />;
+        }
+        if (numOfSelectedItems === 0 && !isFolderPresent) {
+            return <FormattedMessage {...messages.includeSubfoldersNoFoldersToSelect} />;
+        }
+        return <FormattedMessage {...messages.includeSubfoldersDefaultInformation} />;
+    };
```

Comment:
we tend to avoid multiple returns like this unless it's like a guard clause.

could we do something like:

```
let tooltipMessage = messages.includeSubfoldersDefaultInformation;

if (numOfSelectedItems > 1) {
    tooltipMessage = messages.includeSubfoldersMultipleFoldersSelected;
}
if (numOfSelectedItems === 1 && noFoldersSelected) {
    tooltipMessage = messages.includeSubfoldersNoFoldersSelected;
}
if (numOfSelectedItems === 0 && !isFolderPresent) {
    tooltipMessage = messages.includeSubfoldersNoFoldersToSelect;
}

[...]

<Tooltip text={<FormattedMessage {...tooltipMessage} />} />
```